xc7/utils/prjxray_create_edges.py

```diff
@@ -1057,9 +1056,15 @@ def mark_track_liveness(
 
     print('{} Track aliveness committed'.format(now()))
 
-    print("{}: Build channels".format(datetime.datetime.now()))
-    build_channels(conn, pool, alive_tracks)
-    print("{}: Channels built".format(datetime.datetime.now()))
+    write_cur.execute("""BEGIN EXCLUSIVE TRANSACTION;""")
```

Comment:
Should these index commands be part of the initial table SQL file? Or does it actually make sense from a performance point of view to only create them at this point?

Reply:
Exactly.  In general, deferring index creation means there are no surprise rebuilding.  There is a possibly a way to supress that behavior, but this works.